Source/Core/VideoBackends/OGL/main.cpp

```diff
@@ -140,7 +140,7 @@ static void InitBackendInfo()
 	g_Config.backend_info.bSupportsGeometryShaders = true;
 	g_Config.backend_info.bSupports3DVision = false;
 	g_Config.backend_info.bSupportsPostProcessing = true;
-	g_Config.backend_info.bSupportsPaletteConversion = false;
+	//g_Config.backend_info.bSupportsPaletteConversion = true; // is GPU dependent and must be set in renderer
```

Comment:
Shouldn't this documentation go beside the actual variable definition? Then you can just remove this code entirely.